packages/react-native-sdk/src/embed/index.tsx

```diff
@@ -149,6 +232,82 @@ const WherebyEmbed = React.forwardRef<React.ElementRef<typeof WebView>, WherebyE
             }
         });
 
+        const handleMessage = React.useCallback(
+            (event: WebViewMessageEvent) => {
+                const data = JSON.parse(event.nativeEvent.data);
+                if (data.type && WHEREBY_EVENT_TYPES.includes(data.type)) {
+                    props.onWherebyMessage?.(data);
```

Comment:
I'm wondering if this should be moved one line above. This way, when a new event is added to core but not yet supported in react native, customers can still listen to it (unless we don't want to). 

My understanding is: 
- Fully supported events: customers can directly use `onSpecificEvent` callbacks
- New Whereby events (nut not yet supported in React Native): customers can listen to onWherebyMessage and parse the data themselves
- Other (non whereby events): customers can for instance inject some javascript code and listen any kind of messages (for instance: getUserMedia) 

What do you think?

Reply:
no - because this is typed (using the `WherebyEvent` type). If you see the usage of `onWherebyMessage` in the example app, you see that it's typed. Moving it up a line here would break the typing, as it would capture all events, not only whereby specific ones. And we can use `onMessage` for that.

* Fully supported events: customers can directly use `onSpecificEvent` callbacks, **or** `onWherebyMessage`. The benefit here is that you can choose to only listen one or two specific ones, or you can listen to all of them.
* New Whereby events and other (non whereby events): Will both end up in `onMessage`. We can't separate them as Whereby events before it's added to the types here.

Reply:
Understood, thanks for the clarification 👍